openmc/universe.py

```diff
@@ -699,6 +699,33 @@ def auto_geom_ids(self, val):
     def auto_mat_ids(self):
         return self._auto_mat_ids
 
+    @property
+    def material_names(self):
+        """Return the names of the materials that are contained within the
```

Comment:
For `@property`s, you need to add the description in the "Attributes" section of the class docstring for it to show up in the documentation.

Reply:
Sorry about that, I've moved them as suggested 👍